src/Base/StlFeatures.h

```diff
@@ -72,6 +73,10 @@ make_unique(std::size_t n)
 template<class T, class... Args>
 inline typename Detail::UniqueIf<T>::KnownBound
 make_unique(Args&&...) = delete;
+#else
+// Visual Studio 2015 implements make_unique
```

Comment:
The only two versions of VS that support C++11 are 2013 and 2015.  According to MSDN, they both implement make_unique.  I have only actualy tried compiling with Visual Studio 2015 however.